fhir-audit/src/main/java/com/ibm/fhir/audit/logging/impl/WhcAuditCadfLogService.java

```diff
@@ -69,22 +69,22 @@
     private boolean isEnabled = false;
 
     private static final Map<String, Action> fhir2CadfMap = new HashMap<String, Action>() {
-
         private static final long serialVersionUID = 1L;
```

Comment:
do we need a more valid versionUID?

Reply:
this is not something i changed.  paul had a similar comment on a different class, but i'm "meh"